scenes/player/animation_player.gd

```diff
@@ -25,24 +25,18 @@ func _process_walk_idle(_delta: float) -> void:
 
 
 func _process_fighting(_delta: float) -> void:
-	if current_animation in FIGHT_ANIMATIONS and current_animation_position > 0.3:
+	if current_animation == &"blow" and current_animation_position > BLOW_ANTICIPATION_TIME:
 		cancellable = true
 
 	if not player_fighting.is_fighting:
 		if (
 			cancellable
-			and current_animation in FIGHT_ANIMATIONS
-			and current_animation_position <= 0.3
+			and current_animation == &"blow"
+			and current_animation_position <= BLOW_ANTICIPATION_TIME
 		):
 			stop()
 		return
 
-	var angle: float = player.last_nonzero_axis.angle()
-	if abs(-PI / 2 - angle) <= PI / 4:
-		play(&"blow_up")
-	elif abs(PI / 2 - angle) <= PI / 4:
-		play(&"blow_down")
-	elif abs(PI - angle) <= PI / 4:
-		play(&"blow_left")
-	elif abs(angle) <= PI / 4:
-		play(&"blow_right")
+	if current_animation != &"blow":
```

Comment:
All this animation logic seems a bit confusing. Maybe some comments can be helpful to understand what's going on?